examples/target_group_only/.happy/terraform/envs/rdev/main.tf

```diff
@@ -0,0 +1,97 @@
+module "stack" {
+  source = "git@github.com:chanzuckerberg/happy//terraform/modules/happy-stack-eks?ref=heathj/target-group-only"
+
+  image_tag        = var.image_tag
+  image_tags       = jsondecode(var.image_tags)
+  stack_name       = var.stack_name
+  deployment_stage = "rdev"
+  stack_prefix     = "/${var.stack_name}"
+  k8s_namespace    = var.k8s_namespace
+
+  services = {
+    tgonly = {
+      name         = "tgonly"
+      service_type = "TARGET_GROUP_ONLY" // tells happy to only make a target group for your service, not an ALB too (the default behavior)
+
+      path              = "/mypath"        // path to attach the target group of the ALB below
+      health_check_path = "/mypath/health" // the healthcheck route should be below the path specified
+      port              = 3000             // port of the service (see app.js)
+
+      // the external ALB and listener to attach to
+      alb = {
+        name          = aws_lb.this.name
+        listener_port = aws_lb_listener.this.port
+      }
+      platform_architecture = "arm64"
+    }
+  }
+}
+
+# !!!pretend the below code (ALB, listeners, and security group) was created elsewhere!!!
```

Comment:
see comment. this is just for documentation purposes and to make a real working example. In reality, you'd already have these values from your old infrastructure.